common/src/main/java/com/lightstep/opentelemetry/common/VariablesConverter.java

```diff
@@ -183,4 +240,74 @@ static String getHostName() {
       return "";
     }
   }
+
```

Comment:
Any reason to not have this class a top-level one? i.e. not `static class` here ;)

Reply:
just to avoid getter for all fields.

Reply:
sounds good!